README.md

```diff
@@ -104,14 +118,23 @@ For reference, here is a truth table for the expression A && !B. You can fill o
 
 | A     | B     | !B    | A && !B | !A && !B |
 | ----- | ----- | ----- | ------- | -------- |
-| true  | true  | false | false   |          |
-| false | true  | false | false   |          |
-| true  | false | true  | true    |          |
-| false | false | true  | false   |          |
+| true  | true  | false | false   |  false   |
+| false | true  | false | false   |  false   |
+| true  | false | true  | true    |  false   |
+| false | false | true  | false   |  true    |
 
 ## Problem Five
 
 Create a truth table for the expression !(A || B).
+ 
+| A     | B     | !B    | A || B  | !(A || B)|
+| ----- | ----- | ----- | ------- | -------- |
+| true  | true  | false | true    |  false   |
+| false | true  | false | true    |  false   |
+| true  | false | true  | true    |  true    |
```

Comment:
!(true || false) should evaluate to false

Reply:
This has been corrected.